impeller/entity/contents/text_contents.h

```diff
@@ -30,6 +30,10 @@ class TextContents final : public Contents {
 
   void SetColor(Color color);
 
+  /// @brief Force the text color to apply to the rendered glyphs, even if those
+  ///        glyphs are bitmaps.
```

Comment:
nit: add note that this is a trick to get color glyphs behaving correctly for the mask blur case?

Reply:
Done